<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HotKeyWidget</class>
 <widget class="QWidget" name="HotKeyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="buttonChange">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Поменять Горячую Клавишу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonExit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGetHotKey">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ввести Символ</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>120</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkCtrl">
      <property name="text">
       <string>CTRL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkShift">
      <property name="text">
       <string>SHIFT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkAlt">
      <property name="text">
       <string>ALT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
